tests/docker/start.globus.run.tests.ubuntu.sh

```diff
@@ -1,6 +1,11 @@
 #! /bin/bash
+#cd /irods_packages
+#apt install -y ./irods-dev_4.3.3-0~jammy_amd64.deb ./irods-runtime_4.3.3-0~jammy_amd64.deb ./irods-icommands_4.3.3-0~jammy_amd64.deb
```

Comment:
Is this needed?

Reply:
No, it was left over from a previous iteration.  Removed.